tasks/synopsys-detect-task/.taskkey

```diff
@@ -0,0 +1 @@
+cbcc2e69-cf2c-44bd-b6f2-cfa57d33ce4a
```

Comment:
Another hidden file with the same name, but different path. Should there be multiple `.taskkey` files and should they be committed?